LICENSE.txt

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
I put MIT in the github metadata, we might want to align the two. I tend to prefer MIT because it is an even shorter license but I am fine with either.

Reply:
We used this BSD 3 license file in previous projects (you first proposed it for sklearn-numba-dpex). Do you want to put everything on MIT or is there a reason to use MIT rather than BSD 3 for this particular repo ? I just put BSD 3 for consistency in all our projects.

Reply:
Changing the license of an existing project with many contributors is not easy and would be mostly useless (e.g. for scikit-learn).

But since MIT is simpler, it's getting more popular and is the default choice on github. This is why I tend to choose MIT for new repos.

I think nobody cares about the extra clause of the 3-clause BSD license and nobody plans to enforce it in practice. There is already trade-mark right to protect the user of Inria's name. No need for an extra clause in the license.

If you are worried someone might promote product or services that rely on sklearn-engine-benchmark using the name "Franck Charras" and don't want that to happen, then off-course I am fine with using 3-clause BSD :)

Reply:
But just keep in mind to change the project metadata on github so that we have consistent licensing info everywhere **within** this repo.

Reply:
Ok then, I'll revert the license change, and will udpate accordingly our other repos next time I work on it